src/utils/pause/frames/displayName.js

```diff
@@ -19,7 +19,7 @@ const annonymousProperty = /([\w\d]+)\(\^\)$/;
 
 export function simplifyDisplayName(displayName: string) {
   // if the display name has a space it has already been mapped
-  if (/\s/.exec(displayName)) {
+  if (!displayName || /\s/.exec(displayName)) {
```

Comment:
should we update the type `simplifyDisplayName(displayName: string)`?

Reply:
seems like it accepts a string, but then we check if there is no string. I wonder if we should do

`simplifyDisplayName(displayName: string | void)`

Reply:
oh, okay, I'll do that